src/Symfony/Component/ErrorHandler/ErrorRenderer/CliErrorRenderer.php

```diff
@@ -11,8 +11,8 @@
 
 namespace Symfony\Component\ErrorHandler\ErrorRenderer;
 
+use Symfony\Component\Console\Terminal;
 use Symfony\Component\ErrorHandler\Exception\FlattenException;
-use Symfony\Component\VarDumper\Cloner\VarCloner;
 use Symfony\Component\VarDumper\Dumper\CliDumper;
```

Comment:
This `use` statement is not needed anymore, and neither is the `class_exists` below.